lib/rules/link-text.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "link-text",
+  "selector": "a[href]",
+  "tags": [
+    "wcag2a",
+    "wcag244",
+    "cat.text-alternatives",
+    "best-practice"
```

Comment:
This should be tagged with `best-practice` only and not the WCAG stuff. `cat.text-alternatives` is ok.